Mathlib/Algebra/GroupWithZero/NonZeroDivisors.lean

```diff
@@ -277,3 +275,78 @@ lemma nonZeroSMulDivisors_mulOpposite_eq_op_nonZeroDivisors :
     Rᵐᵒᵖ ⁰[R] = R⁰.op := rfl
 
 end nonZeroSMulDivisors
+
+noncomputable section Associated
+
+open nonZeroDivisors
+
+/-- The units of the monoid of non zero divisors of `α` are equivalent to the units of `α`. -/
+def nonZeroDivisorsUnitsEquiv (α : Type*) [MonoidWithZero α] :
+    (α⁰)ˣ ≃* αˣ :=
```

Comment:
```suggestion
variable (α : Type*) [MonoidWithZero α]

def nonZeroDivisorsUnitsEquiv : (α⁰)ˣ ≃* αˣ :=
```
and below. Also does `α⁰ˣ` work as notation? Maybe it's confusing.

Reply:
It does work and I do think it's that confusing...